livingdoc-repositories/src/main/kotlin/org/livingdoc/repositories/config/Configuration.kt

```diff
@@ -0,0 +1,20 @@
+package org.livingdoc.repositories.config
+
+data class Configuration(
+        var repositories: List<RepositoryDefinition> = emptyList()
+) {
+
+    companion object {
+
+        fun load(): Configuration {
+            return loadFromClassPathFile("livingdoc.yml")
+        }
+
+        fun loadFromClassPathFile(filePath: String): Configuration {
```

Comment:
I'd turn this into `loadFromStream` and keep the knowledge about files and classpaths in `load()`. That gives us more options later and also would allow to specify yaml directly in the test file.